server/elasticsearch/src/main/java/org/hawkular/apm/server/elasticsearch/SpanServiceElasticsearch.java

```diff
@@ -91,6 +92,16 @@ public Span getSpan(String tenantId, String id) {
         if (!response.isSourceEmpty()) {
             try {
                 span = deserialize(response.getSourceAsString(), Span.class);
+
+                /**
+                 * Enrich server span with client span annotations
+                 */
+                if (span.serverSpan() && span.url() == null) {
+                    Span clientSpan = getSpan(tenantId, SpanUniqueIdGenerator.getClientId(span.getId()));
+                    List<BinaryAnnotation> binaryAnnotations = new ArrayList<>(clientSpan.getBinaryAnnotations());
```

Comment:
How do you identify which annotation is the URL? We don't want this to be http specific.